pykern/pkinspect.py

```diff
@@ -126,10 +126,13 @@ def caller_module():
 
     Will raise exception if calling from __main__
 
+    Args:
+        exclude_first (bool): skip first module found [True]
```

Comment:
calling module, not the caller's caller.

Reply:
I'm not sure what your saying?

Reply:
The documentation is unclear. exclude_first means to not do what the documentation says it is supposed to do so the documentation should read something like:

```txt
    """Returns `Call` for caller's caller with certain exclusions

    Will not return the same module as the calling module, that is,
    will iterate until a new module is found, when `exclude_first` is
    truthy. If `ignore_modules` is defined, will skip over all call frames
    for those modules as well.

    Note: may return ``__main__`` module.

    Will raise exception if calling from ``__main__``

    Args:
        ignore_modules (list): other modules (objects) to exclude [None]
        exclude_first (bool): skip over calls from calling module [True]

    Returns:
        Call: keys: filename, lineno, name, module
```

Reply:
fix later, because both functions need updating, and better to get this out the door.